### DISCUSSION TITLE
Custom Death Messages

### DISCUSSION THREAD

**Discussion by ItLucifer-MRPG** [Zrips/CMI#5652] [Feature Requests]
Ive seen CMI has custom join and leave message, but i feel like built in custom death messages would be nice, its basically the only thing that looks vanilla while everything else is customized

**Comment by mrfloris** [comment]
Issues is now used for Bug reports, Discussions is used for feature requests;
https://github.com/Zrips/CMI/discussions

**Comment by Zrips** [comment]
Work in progress

**Comment by mrfloris** [comment]
Awesome! :D
